packages/nimble-components/src/button/tests/button.mdx

```diff
@@ -16,3 +16,32 @@ import * as buttonStories from './button.stories';
 <Canvas of={buttonStories.outlineButton} />
 <Controls of={buttonStories.outlineButton} />
 <Stories of={buttonStories} />
+
+# Usage Docs
+
+<br />
+
+## Sizing
+
+    The label button has two sizes: 32px and 24px, while the icon only button has two smaller sizes: 18px and 16px.
```

Comment:
I don't think we use the term "label button" anywhere else. Doesn't immediately make me think of the non-icon button. I would just say "standard button".